ecosystem/indexer-grpc/indexer-grpc-utils/src/cache_operator.rs

```diff
@@ -6,10 +6,8 @@ use anyhow::Context;
 use redis::{AsyncCommands, RedisError, RedisResult};
 
 // Configurations for cache.
-// The cache size is estimated to be 3M transactions.
-// For 3M transactions, the cache size is about 25GB.
-// At TPS 20k, it takes about 2.5 minutes to fill up the cache.
-const CACHE_SIZE_ESTIMATION: u64 = 3_000_000_u64;
+// If the entry doesn't exist, fallback to file store.
+const CACHE_SIZE_ESTIMATION: u64 = 500_000_u64;
```

Comment:
Should we make this even smaller? like 100 or 200k? What's the cost implication of 500k?